plugins/dfu/fu-dfu-device.c

```diff
@@ -804,15 +804,21 @@ fu_dfu_device_request_detach(FuDfuDevice *self, FuProgress *progress, GError **e
 	FuDfuDevicePrivate *priv = GET_PRIVATE(self);
 	GUsbDevice *usb_device = fu_usb_device_get_dev(FU_USB_DEVICE(self));
 	const guint16 timeout_reset_ms = 1000;
+	guint8 force_detach = 0x00;
```

Comment:
maybe `guint16 iface_number = priv->iface_number` and then in the conditional `iface_number |= 0x01u << 8`

Reply:
works for me